### PR TITLE
emoji fallback renderer

### PR DESCRIPTION
**Pull request by julianseeger**
makes sure threre is always something rendered (emoji or emoji with forced fitzpatrick modifier or at least the unicode emoji)

### CODE REVIEW

**Comment by audax** [comment]
No test? 🍡

**Comment by julianseeger** [comment]
the code is coupled to static methods ... not really testable :'(

File: client/src/main/java/de/qabel/desktop/ui/actionlog/item/renderer/partial/EmojiRenderer.java [hunk 1]
```diff
@@ -36,11 +40,21 @@ protected Node replace(String match, Matcher matcher) {
...
             return new Text(match);
         }
-        ImageView icon = Icons.getIcon(EmojiManager.getForAlias(match), 24);
-        icon.getStyleClass().add("emoji");
-        Label emojiLabel = new Label("", icon);
-        emojiLabel.getStyleClass().add("emoji-container");
-        emojiLabel.setPadding(new Insets(0, 1, 0, 1));
-        return emojiLabel;
+        try {
+            ImageView icon = Icons.getIcon(EmojiManager.getForAlias(match), 24);
```
Comment (audax) [inline client/src/main/java/de/qabel/desktop/ui/actionlog/item/renderer/partial/EmojiRenderer.java]: extract & override in Test

**Comment by julianseeger** [comment]
test added in https://github.com/Qabel/qabel-desktop/pull/562/commits/a6d2746a695e92a5a9a3b2c62888fb90ff35f6fa

**Pull request merged**